code/src/App.js

```diff
@@ -1,9 +1,87 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+
+import ThoughtsList from './components/ThoughtsList';
+import NewThoughts from './components/NewThoughts';
+
+import { API_URL, LIKES_URL } from './reusable/urls';
 
 export const App = () => {
+  const [thoughtsList, setThoughtsList] = useState([]);
+  const [newMessage, setNewMessage] = useState('');
+
+  useEffect(() => {
+    fetchThoughtsList();
+  }, []);
+
+  const fetchThoughtsList = () => {
+    fetch(API_URL)
+      .then(res => res.json())
+      .then(messages => setThoughtsList(messages))
+      .catch(err => console.error(err));
+  }
+
+  const onNewMessageChange = (event) => {
+    setNewMessage(event.target.value);
+}
+
+  const onFormSubmit = (event) => {
+    event.preventDefault();
+
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ message: newMessage })
+    };
+    
+  fetch(API_URL, options)
+    .then(res => res.json())
+    .then(() => {
+      window.location.reload()
```

Comment:
It is an anti-pattern to reload single page application. You need to rethink the state/props structure to make your app re-render